app/routes/pages/components/Sidebar.tsx

```diff
@@ -45,9 +45,6 @@ class Sidebar extends Component<Props, State> {
                 <Icon name="close" size={50} />
               </button>
               <h3 className={styles.sidebarHeader}>Om Abakus</h3>
-              <h4 className={styles.sidebarSubtitle}>
-                {categorySelected ? categorySelected : 'Generelt'}
-              </h4>
```

Comment:
Good change. This has annoyed me for a long time.

But, remember to remove the css class from `Sidebar.css` as well.

Reply:
Nice one, removed now!